src/JPEGView/ImageLoadThread.cpp

```diff
@@ -15,6 +15,7 @@
 #include "PNGWrapper.h"
 #include "JXLWrapper.h"
 #include "HEIFWrapper.h"
+#include "WEBPWrapper.h"

```

Comment:
I think webp will be supported in XP still.  Compiling it takes a different set of flags, but I will do this testing.  Thanks for pulling in the latest commits so it's easier to merge 😅

I might also move these into the ImageLoadThread.h, but I go back and forth on whether that include styling is better ...